src/main/kotlin/tanvd/grazi/grammar/GrammarChecker.kt

```diff
@@ -88,6 +88,16 @@ class GrammarChecker(private val ignoreChar: LinkedSet<(CharSequence, Char) -> B
                     val newRange = IntRange(typo.location.range.start + startShift - range.start, typo.location.range.endInclusive + endShift - range.start)
                     typo.copy(location = typo.location.copy(range = newRange, pointer = firstToken.toPointer()))
                 } else null
+            }?.let { typo ->
```

Comment:
Ok, it is definitely should not be here. Grammar checker is an abstract checker that works for all languages. And here is a specific fix for specific rule of English. Think about moving it to some global `LangToolFixes` that will provide fixes for language-tool lang rules that we consider broken.

Reply:
Done